### PR TITLE
token deployment flow

### LINKED ISSUE #497: Token deployment frontend
Need to create this two extra pages in the DAO creator to include the option to deploy a token:

https://xd.adobe.com/view/61a546b6-c405-41c8-a8a3-793ca84992e7-34f6/screen/2c9a1970-2471-4fc5-89d8-a0b31308e29f?fullscreen

Manank will additionally add the code from FA2 bakery into this PR so we can deploy the feature intact.

### LINKED ISSUE #550: Fix Token Field Validation in DAO Creator
When you paste the Token address, it says it can't find it, even though it's a valid address.

### LINKED ISSUE #564: Bugs: Token deployer
**Description (what needs to be done) / Expected Behavior**

- [x] Bug: validation error persists after entering token address
- [x] 
- [x] you don't need to have an address. If no address is inputted, the entire amount is minted to the address of the user who is logged in
- [x] 
- [x] User experience issue: As a person deploying a token for the first time, at first in this screen there is a placeholder that says Token ID, but as soon as I paste the token address, that changes to 0 without a label. As a new user I wouldn't understand what that 0 is. I'd suggest we add labels underneath the fields, since about 10 users over time have asked us before what the token ID is.
- [x] 
- [x] Token deployer: address fields are too narrow for full address to show. One solution is to show first part of address and ... last part of address.

**Acceptance Criteria**
validation errors listed above are addressed. Labels below entry boxes on token address, ID, FA2. Please check for any further issues in token deployer.

**Supporting information (Figma link, notes, etc)**

**When finished, please add Link to the PR or completed Figma / design asset, as a comment to the ticket.**

### PR DESCRIPTION
**Pull request by fabiolalombardim**
Closes #497 
Closes #550 
Closes #564

### CODE REVIEW

**Comment by fabiolalombardim** [comment]
Closes #512

File: src/modules/creator/steps/Template.tsx [hunk 1]
```diff
@@ -156,10 +114,26 @@ export const Template = (): JSX.Element => {
...
+          container
+          direction="column"
+          justifyContent="flex-start"
+          alignItems="center"
+          xs={isMobileSmall ? 12 : 6}
+          onClick={() => update("lite")}
+          className={selectedTemplate === "lite" ? style.selected : ""}
+        >
+          <LiteIcon style={{ marginBottom: 14 }} />
+          <BoxTitle color="textSecondary">Lite DAO</BoxTitle>
```
Comment (Man-Jain) [inline src/modules/creator/steps/Template.tsx:134]: The lite dao shouldn't be here

File: src/modules/explorer/components/ProposalForm.tsx [hunk 2]
```diff
@@ -85,7 +85,8 @@ const enabledForms: Record<
       component: ({ open }) => <UpdateRegistryDialog open={open} />
     }
   ],
-  "": []
+  "": [],
+  "lite": []
```
Comment (Man-Jain) [inline src/modules/explorer/components/ProposalForm.tsx:89]: same here

File: src/modules/creator/state/types.ts [hunk 3]
```diff
@@ -66,7 +66,7 @@ export interface NavigationBarProps {
   }
 }
 
-export type DAOTemplate = "lambda" | ""
+export type DAOTemplate = "lambda" | "lite" | ""
```
Comment (Man-Jain) [inline src/modules/creator/state/types.ts:69]: LIte shouldn't be here

**Review by Man-Jain — CHANGES_REQUESTED** [review]
Screenshot 2023-04-21 at 3 38 03 PM

Can we redirect user directly to the Step two instead of showing this, this is a big unnecessary.

**Comment by Man-Jain** [comment]
Screenshot 2023-04-21 at 3 41 56 PM

Let's put a label besides the Token Address and make it super clear that this is the token Address 

Something Like 

Your Token Address - KT1A4bpidDyfsSBzac1tka7U9BS65FCK3HrG

**Comment by Man-Jain** [comment]
Initial token distribution
- Should not be able to allocate tokens to the same address multiple times
- On allocating more than the total supply, the page hangs and you cannot change the fields

**Review by Man-Jain — APPROVED** [review]

**Pull request merged**